ember-intl/addon/helpers/format-date.ts

```diff
@@ -14,7 +14,7 @@ interface FormatDateSignature {
     Named?: Options & { allowEmpty?: boolean };
     Positional: [Value?, Options?];
   };
-  Return: string | undefined;
+  Return: string;
```

Comment:
Technically, I'm fixing a regression here, because the return type had been `string` for all helpers in `v6.1.0`.

https://github.com/ember-intl/ember-intl/blob/v6.1.0/addon/helpers/-format-base.d.ts#L13